sympy/series/formal.py

```diff
@@ -972,6 +982,20 @@ def infinite(self):
 
         return self.ind + inf_sum
 
+    @property
+    def variables(self):
+        """Returns a tuple of variables that are bounded"""
+        return ()
+
+    @property
+    def free_symbols(self):
+        """
+        This method returns the symbols in the object, excluding those
+        that take on a specific value (i.e. the dummy symbols).
+        """
+        return (set(j for i in self.args for j in i.free_symbols)
+                .difference(self.variables))
```

Comment:
Why re-implementing this? It's already part of ``SeriesBase``. Note ``FormalPowerSeries`` inherits from it.

Reply:
Sorry. I kept that for checking something. Forgot to remove it. Thanks @leosartaj